.github/workflows/release-nym-vpn-desktop.yml

```diff
@@ -41,15 +41,15 @@ jobs:
         os:
           [
             ubuntu-22.04-16-core,
-            custom-runner-mac-m1,
+            macos-14,
           ]
     runs-on: ${{ matrix.os }}
     env:
       WG_GO_LIB_PATH: ${{ github.workspace }}/lib
 
     steps:
       - name: "Cleanup working directory"
-        if: contains(matrix.os, 'custom')
+        if: contains(matrix.os, 'macos')
```

Comment:
Don't need to do any cleanup on those runners, they are supposed to be clean on each run (I hope)